app/lib/constants/role.rb

```diff
@@ -1,27 +1,33 @@
 module Constants
   module Role
-    BASE_ROLES = ["Warn",
-                  "Comment Suspend",
-                  "Suspend",
-                  "Regular Member",
-                  "Trusted",
-                  "Pro"].freeze
+    BASE_ROLES = [
+      "Warn",
+      "Comment Suspend",
+      "Suspend",
+      "Regular Member",
+      "Trusted",
+      "Pro",
+    ].freeze
 
-    SPECIAL_ROLES = ["Admin",
```

Comment:
Just moved the first and last brackets in their own line, and somewhat alphabetized the list.